### PR TITLE
Support real GDB

### PR DESCRIPTION
**Pull request by daym**
Fixes <https://github.com/emacs-lsp/dap-mode/issues/790>.

### CODE REVIEW

File: dap-gdb.el [hunk 1]
```diff
@@ -0,0 +1,89 @@
+;;; dap-gdb.el --- Debug Adapter Protocol mode for LLDB/GDB      -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2019  Ivan Yonchovski
+;; Copyright (C) 2024  Danny Milosavljevic
+
+;; Author: Ivan Yonchovski yyoncho@gmail.com
```
Comment (jcs090218) [inline dap-gdb.el:6]: It's fine to leave yourself as the author. 🤔

**Review by jcs090218 — COMMENTED** [review]
Can you update the CHANGELOG too?

**Comment by jcs090218** [comment]
There is a compile error:

```
dap-gdb.el:30:1:Error: Wrong number of arguments: (3 . 3), 2
```

**Comment by daym** [comment]
Ok, fixed a typo and tested for a while with lsp-rust.

**Review by jcs090218 — APPROVED** [review]

**Comment by jcs090218** [comment]
Thank you!

**Pull request merged**